src/main/java/com/training/assignments/Operations.java

```diff
@@ -0,0 +1,99 @@
+package com.training.assignments;
+
+import java.util.HashMap;
+
+public class Operations {
+    public String getCompanyReviews(String companyName) {
+        HashMap<String, String> company = new HashMap<>();
+
+        company.put("Endee engineers", "good company for production of portable and online instrument of gas analyzer");
+        company.put("Aaion automation", "installation of GPS tracker");
+        company.put("Thermolab", "good company for production of stabilizers and incubator");
+        company.put("Shreeji solution and services", "provide solutions of fire alarm system");
+        company.put("Google", "Best platform where u will get solution for everything");
+
+        return company.get(companyName);
+    }
+
+    public String getCompanysFeedback(String company) {
+        switch (company) {
+            case "Endee engineers":
+                return "good company for production of portable and online instrument of gas analyzer";
+
+            case "Aaion automation":
+                return "installation of GPS tracker";
+
+            case "Thermolab":
+                return "good company for production of stabilizers and incubator";
+
+            case "Shreeji solution and services":
+                return "provide solutions of fire alarm system";
+
+            case "Google":
+                return "Best platform where u will get solution for everything";
+
+            default:
+                return "All companies are good";
+        }
+    }
+
+    public String getBehaviour(String people) {
+        HashMap<String, String> behaviour = new HashMap();
+
+        behaviour.put("good", "go to heaven");
+        behaviour.put("very good", "go to heaven with dog");
+        behaviour.put("Bad", "go to hell");
+        behaviour.put("Very Bad", "go to hell");
+
+        return behaviour.get(people);
+    }
+
+    public String getPlaceBehaviourBased(String people) {
+        if (people == "good") {
+            return "go to heaven";
+        } else if (people == "very good") {
+            return "go to heaven with dog";
+        } else if (people == "Bad") {
+            return "go to hell";
+        } else if (people == "Very Bad") {
+            return "go to hell";
+        }
+        return "";
```

Comment:
always use .equals for string rather than == -> it may cause unexpected outputs.